app/src/main/java/com/kylecorry/trail_sense/tools/tools/infrastructure/Tools.kt

```diff
@@ -218,6 +218,7 @@ object Tools {
     const val QUICK_ACTION_WHISTLE = 8
     const val QUICK_ACTION_WHITE_NOISE = 9
     const val QUICK_ACTION_LOW_POWER_MODE = 10
+    const val QUICK_ACTION_SUNRISE_ALERT = 12
```

Comment:
Can you make this 25? 12 was already taken at some point and removed (by a migration script)

Reply:
Fixed